server/src/assetComputeDevTool.js

```diff
@@ -240,7 +240,7 @@ async function setupCloudStorage() {
         );
     }
     else {
-        throw new Error("Cloud storage credentials not set.");
+        throw new Error("Neither AWS nor Azure cloud storage credentials were provided, please see readme for information");
```

Comment:
There is no readme with that information yet, but I think we'll add one, so LGTM.

Reply:
Message: "Neither AWS nor Azure cloud storage credentials were provided, please set credentials up for either AWS or Azure"

Reply:
I think she was referring to this: https://github.com/adobe/asset-compute-devtool#1-s3-bucket-or-azure-blob-storage-credentials

but I agree I think saying something like this is more straightforward, especially considering some developers may see this in the context of our plugin (which wont contain the same readme)

Message: "No cloud storage credentials were provided, please set credentials up for either AWS or Azure"